internal/server/data_source.go

```diff
@@ -91,6 +91,33 @@ func getCommonDataSchema() map[string]*schema.Schema {
 			Computed: true,
 			Elem:     &schema.Schema{Type: schema.TypeInt},
 		},
+		"network": {
+			Type:     schema.TypeSet,
+			Optional: true,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"network_id": {
+						Type:     schema.TypeInt,
+						Required: true,
+					},
+					"ip": {
+						Type:     schema.TypeString,
+						Computed: true,
+						Optional: true,
```

Comment:
For the data source all fields are always set, so you can remove the `Optional: true` from `ip`, `alias_ips` and `mac_address` here and in the docs.